README.md

```diff
@@ -28,16 +28,13 @@ cd infra/
 git clone https://github.com/theforeman/foreman-ansible-modules.git
 ```
 
-Now, add or update your `ansible.cfg` file to include the modules path:
+Note the `ansible.cfg` file cloned with foreman-ansible-modules. The ansible.cfg
+needs to be in your current directory when you run `ansible` or
+`ansible-playbook`. You can copy it to another location or add it to your
+current ansible configuration.
```

Comment:
I would add a sentence that the paths might need updating if copying the file to a different location.